File: compare/basic/sqrt_sum.f90
PROGRAM sqrt_sum
  IMPLICIT NONE
  INTEGER, PARAMETER :: N = 100000000
  INTEGER :: i
  REAL(8) :: sum

  sum = 0.0D0
  DO i = 1, N
    sum = sum + SQRT(REAL(i, 8))
  END DO

  PRINT *, "Sum:", sum
END PROGRAM sqrt_sum

File: compare/basic/sqrt_sum.cpp
#include <cmath>
#include <iostream>

int main() {
  const long N = 100000000;
  double sum = 0.0;
  for (long i = 1; i <= N; ++i) {
    sum += std::sqrt(i);
  }
  std::cout << "Sum: " << sum << std::endl;
  return 0;
}
